manifests/o/OlegShparber/Zeal/0.7.0/OlegShparber.Zeal.installer.yaml

```
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: OlegShparber.Zeal
+PackageVersion: 0.7.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: wix
+Installers:

```

```suggestion
Dependencies:
  PackageDependencies:
   - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

(Automated response - build 728.)